armi/settings/tests/test_settings.py

```diff
@@ -58,7 +58,7 @@ def defineSettings():
         ]
 
 
-class DummyPlugin2(plugins.ArmiPlugin):
+class DummyParamPlugin2(plugins.ArmiPlugin):
```

Comment:
Not sure why the names of these classes changed. At the least, I would have expected them to change into `DummySettingPluginX`, since they define a setting not a parameter.

Reply:
Ah, yeah, that's a funny typo as a result of some other testing I was doing. I'll make that change. 1 sec.

Reply:
Fixed.